app/models/comment.rb

```diff
@@ -143,6 +143,13 @@ def title(length = 80)
     HTMLEntities.new.decode(truncated_text)
   end
 
+  # A helper method to encode logic formerly part of spam detection.
+  def from_recently_registered_user?
```

Comment:
SpamAssassin has a rule called "day old bread" for recently registered domain names that's effectively the same idea.

Reply:
what they have against "day old bread"? 🧐